docs/src/examples.rst

```diff
@@ -14,205 +14,265 @@
-   import cdk = require('@aws-cdk/cdk');
+This example creates a Fargate service,
+which requires a VPC, a cluster, and a task definition.
 
-   class MyStack extends cdk.Stack {
-     constructor(parent: cdk.App, name: string, props?: cdk.StackProps) {
-       super(parent, name, props);
+.. tabs::
 
-       new s3.Bucket(this, 'MyBucket', {
-         bucketName: 'MyS3Bucket',
-         encryption: s3.BucketEncryption.KmsManaged
-       });
-     }
-   }
+    .. group-tab:: TypeScript
 
-   const app = new cdk.App();
+        Add the following import statements:
 
-   new MyStack(app, 'MyStack');
+        .. code-block:: typescript
 
-   app.run()
+            import ec2 = require('@aws-cdk/aws-ec2');
+            import ecs = require('@aws-cdk/aws-ecs');
+
+        Add the following code to the end of the constructor:
+
+        .. code-block:: typescript
+
+            const vpc = new ec2.VpcNetwork(this, 'MyVpc', {
+              maxAZs: 3 // Default is all AZs in region
+            });
+
+            // Create an ECS cluster
+            const cluster = new ecs.Cluster(this, 'MyCluster', {
+              vpc: vpc
+            });
+
+            const taskDefinition = new ecs.FargateTaskDefinition(this, 'MyFargateTaskDefinition', {
+              cpu: '512',  // Default is 256
+              memoryMiB: '2048'  // Default is 512
+            });
+
+            // The task definition for the container.
+            taskDefinition.addContainer('MyContainer', {
+              image: ecs.ContainerImage.fromDockerHub('amazon/amazon-ecs-sample')    // Required
+            });
+
+            new ecs.FargateService(this, 'MyFargateService', {
+              taskDefinition: taskDefinition,  // Required
+              cluster: cluster,  // Required
+              desiredCount: 6,  // Default is 1
+            });
+
+        Save it and make sure it builds and creates a stack.
+
+        .. code-block:: sh
+
+            npm run build
+            cdk synth
+
+        You should see a stack of a couple hundred lines, so we won't show it here.
+        The stack should contain one default instance, a private subnet and a public subnet
+        for the three availability zones, and a security group.
+
+        Deploy the stack.
+
+        .. code-block:: sh
+
+            cdk deploy
+
+        |CFN| displays information about the dozens of steps that
+        it takes as it deploys your app.
+
+.. _creating_ecs_l2_example_4:
+
+Step 4: Adding to the Service
+-----------------------------
+
+Let's add auto-scaling to our cluster.
+
+Add the following code just after you declare a cluster.
+This code creates a cluster that scales out to up to three **t2.xlarge** instances.
+
+.. tabs::
+
+    .. group-tab:: TypeScript
+
+        Add the following import statements:
+
+        .. code-block:: typescript
+
+            .. code-block:: ts
+
+                // Add capacity to it
+                cluster.addDefaultAutoScalingGroupCapacity({
```

Comment:
Sorry, no. This is to add EC2 capacity to run Ec2Services on. You've been creating a FargateService so you don't need to do this.

You want what this section says:

https://github.com/awslabs/aws-cdk/blob/master/packages/%40aws-cdk/aws-ecs/README.md#task-autoscaling